src/wasm/wasm-binary.cpp

```diff
@@ -5293,8 +5287,8 @@ void WasmBinaryBuilder::visitRefFunc(RefFunc* curr) {
   // To support typed function refs, we give the reference not just a general
   // funcref, but a specific subtype with the actual signature.
   // FIXME: for now, emit a nullable type here
-  curr->finalize(
-    Type(HeapType(getFunctionSignatureByIndex(index)), /* nullable = */ true));
+  curr->finalize(Type(HeapType(getSignatureByFunctionIndex(index)),
+                      /* nullable = */ true));
```

Comment:
I think this would be incorrect. The index here is the index of the *function*, hence we call "get signature by function index". And `types` is the list of types by type index.

Reply:
Oh, you're right. Sorry!